data/bedrock/latest/proto.yml

```diff
@@ -1,7 +1,7 @@
 # Created from MiNET and gophertunnel docs
 # The version below is the latest version this protocol schema was updated for.
 # The output protocol.json will be in the folder for the version
-!version: 1.19.63
+!version: 1.19.62
```

Comment:
Just curious, what's the logic about reverting this field? Is it used anywhere or just git log cleanliness?

Reply:
It's used when determining where the protocol.json file will be written (which is ultimately used by protodef)